src/app/LoginUseCaseFactory.java

```diff
@@ -25,11 +25,11 @@ private LoginUseCaseFactory() {}
     public static LoginView create(
             ViewManagerModel viewManagerModel,
             LoginViewModel loginViewModel,
-            SearchViewModel searchViewModel,
+            LoggedInViewModel loggedInViewModel,
```

Comment:
Accurate the implementation

Reply:
we seprate the logged in view and search view so when we log in we first go to logged in then search view